### PR TITLE
add E80-1800 PCB

### PR DESCRIPTION
**Pull request by ebastler**
Add support for the E80-1800 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ebastler/e80_1800/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#define BACKLIGHT_PWM_CHANNEL 2
+#define BACKLIGHT_PAL_MODE 2
+#define BACKLIGHT_LEVELS 5
+#define BACKLIGHT_ON_STATE 1
+#define BACKLIGHT_PWM_OUTPUT_FREQUENCY 1000
+
+/* I2C for OLED */
+#define I2C_DRIVER I2CD2
+#define I2C1_SCL            10
+#define I2C1_SDA            11
```
Comment (drashna) [inline keyboards/ebastler/e80_1800/config.h:60]: Deprecated.

Should be:
```suggestion
#define I2C1_SCL_PIN        B10
#define I2C1_SDA_PIN        B11
```

Assuming B pins, at least.

File: keyboards/ebastler/e80_1800/info.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "keyboard_name": "E80_1800",
+    "url": "https://github.com/ebastler/E80-1800",
+    "maintainer": "ebastler",
+    "width": 19.5,
+    "height": 7.25,
```
Comment (drashna) [inline keyboards/ebastler/e80_1800/info.json:6]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ebastler/e80_1800/config.h [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+#define I2C1_SCL_PAL_MODE   1
+#define I2C1_SDA_PAL_MODE   1
+#define I2C1_TIMINGR_PRESC  0U
+#define I2C1_TIMINGR_SCLDEL 3U
+#define I2C1_TIMINGR_SDADEL 1U
+#define I2C1_TIMINGR_SCLH   3U
+#define I2C1_TIMINGR_SCLL   9U
+
+/* Set the OLED timeout to 30 s to prevent burn-in */
+#define OLED_TIMEOUT 30000
```
Comment (drashna) [inline keyboards/ebastler/e80_1800/config.h:70]: Just a heads up, this is defined by default if a kb/user define isn't set, and is set to 60000 by default.  

If you  want to ensure that you prevent burnout, this isn't needed.

File: keyboards/ebastler/e80_1800/readme.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# E80-1800
+
+![E80_1800_sample_image](https://github.com/ebastler/E80-1800/blob/master/img/bottom.png)
```
Comment (drashna) [inline keyboards/ebastler/e80_1800/readme.md:3]: please rehost this on an image hosting site, such as imgur

File: keyboards/ebastler/e80_1800/config.h (hunk 3 above)
Comment (ebastler) [reply keyboards/ebastler/e80_1800/config.h]: Thanks for the heads-up, I was not aware of this. I removed the lines.

File: keyboards/ebastler/e80_1800/readme.md (hunk 4 above)
Comment (ebastler) [reply keyboards/ebastler/e80_1800/readme.md]: Okay, changed to imgur

File: keyboards/ebastler/e80_1800/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# E80-1800
+
+![E80_1800_sample_image](https://i.imgur.com/yzCfVSt.png)
```
Comment (fauxpark) [inline keyboards/ebastler/e80_1800/readme.md:3]:
```suggestion
![E80_1800_sample_image](https://i.imgur.com/yzCfVSth.png)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**